packages/hash/external-services/kratos/kratos.dev.yml

```diff
@@ -1,6 +1,6 @@
 # Based on the quick-start example https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml
```

Comment:
Don't we need to set `can_interrupt: true` for the registration-flow web-hook, as specified in [this example](https://github.com/ory/docs/pull/1142/files)? Otherwise to my understanding the kratos identity is created whether the user entity was successfully created in the graph or not.

Reply:
Whoopsie, must have been removed when i rebased. You're right!

Reply:
4a16d43d0d05558ce3f9e1aae2b78d98ef60a3da